Curve-finance-suite/Remove-Imbalace-Liquidity/src/agent.ts

```diff
@@ -0,0 +1,53 @@
+import { ethers } from 'ethers';
+import { 
+  Finding, 
+  HandleTransaction, 
+  FindingSeverity, 
+  FindingType 
+} from 'forta-agent'
+import {
+  decodeParameters,
+  FindingGenerator,
+  provideEventCheckerHandler,
+} from "forta-agent-tools";
+
+export const CONTRACT_ADDRESS = '0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7';
```

Comment:
This is one specific pool, we may run this agent for more pools too.

Reply:
Let's use a list of contract addresses here ( from `https://curve.readthedocs.io/ref-addresses.html#base-pools` for example) or specify in the `README` that this is only checking `StableSwap3Pool`.